index.d.ts

```diff
@@ -0,0 +1,12 @@
+import {CLIEngine} from 'eslint';
+
+/**
+Format the lint results.
+
+@param results - Lint result for the individual files.
+@param data - Extended information related to the analysis results.
```

Comment:
The description could be more specific about what kind of extended information it expects.

Reply:
Fixed by new @types/eslint package